herd/tests/diycross/AArch64.MTE/MTE.cfg

```diff
@@ -1,4 +1,4 @@
-variant memtag
+variant memtag,async,handled
```

Comment:
I am wondering with `async` as MTE mismatch control, wouldn't the fault specification be irrelevant?

Reply:
That's a good point, maybe it doesn't make sense. I am happy to remove handled.

However, maybe it's worth pointing out that this PR changes the default behaviours. Previously, the default precision was handled. This meant asynchronous MTE (no exceptions on tag mismatches) and handled (retry the instruction) for MMU faults. I've changed the defaults to synchronous MTE and fatal for MMU faults, as I think these defaults are more frequently used.

The primary reason for above change was to ensure that the previously default behaviour is retained.

Reply:
> I've changed the defaults to synchronous MTE and fatal for MMU faults, as I think these defaults are more frequently used.

I agree that the new defaults are better. My local test suite needs a few adaptations. But again I agree.